pyodide-build/pyodide_build/tests/test_buildall.py

```diff
@@ -52,7 +52,9 @@ def from_yaml(cls, path):
 
 
 def test_generate_repodata(tmp_path):
-    pkg_map = buildall.generate_dependency_graph(PACKAGES_DIR, {"pkg_1", "pkg_2"})
+    pkg_map = buildall.generate_dependency_graph(
+        PACKAGES_DIR, {"pkg_1", "pkg_2", "libtest"}
```

Comment:
I assume adding libtest here demonstrates the bug you fixed? Why?

Reply:
No, this one is not related to the bug. I added this to check that static libraries are not included in the `repodata.json`. which was not covered by previous tests.